src/lib/homestore_backend/hs_homeobject.cpp

```diff
@@ -105,14 +105,16 @@ void HSHomeObject::register_homestore_metablk_callback() {
     HomeStore::instance()->meta_service().register_handler(
         _shard_meta_name,
         [this](homestore::meta_blk* mblk, sisl::byte_view buf, size_t size) { on_shard_meta_blk_found(mblk, buf); },
-        [this](bool success) { on_shard_meta_blk_recover_completed(success); }, true);
+        [this](bool success) { on_shard_meta_blk_recover_completed(success); }, true,
+        std::optional< meta_subtype_vec_t >(meta_subtype_vec_t{_pg_meta_name}));
```

Comment:
Do we really have to wrap all these arguments with `std::optional<...>`, won't the compiler deduce this type automatically from the function signature?